src/components/FooterTodo/FooterTodo.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { StatusToFilter } from '../../utils/StatusToFilter';
+import { TodoFilter } from '../TodoFilter';
+
+interface Props {
+  filterStatus: string,
+  onFilterStatusChange: (newStatus: StatusToFilter) => void,
+}
+
+export const FooterTodo: React.FC<Props> = React.memo(({
+  filterStatus, onFilterStatusChange,
+}) => {
+  return (
+    <footer className="todoapp__footer" data-cy="Footer">
+      <span className="todo-count" data-cy="todosCounter">
+        4 items left
```

Comment:
use here the number of how much filtered todos is left

Reply:
yeap, i will do it at the next part of solution